docs/guides/guide-for-latex-users.md

```diff
@@ -75,21 +75,23 @@ Emphasis (usually rendered as italic text) is expressed by enclosing text in
 Here is a list of common markup commands used in LaTeX and their Typst
 equivalents. You can also check out the [full syntax cheat sheet]($syntax).
 
-| Element          | LaTeX                     | Typst                  | See        |
-|:-----------------|:--------------------------|:-----------------------|:-----------|
-| Strong emphasis  | `\textbf{strong}`         | `[*strong*]`           | [`strong`] |
-| Emphasis         | `\emph{emphasis}`         | `[_emphasis_]`         | [`emph`]   |
-| Monospace / code | `\texttt{print(1)}`       | ``[`print(1)`]``       | [`raw`]    |
-| Link             | `\url{https://typst.app}` | `[https://typst.app/]` | [`link`]   |
-| Label            | `\label{intro}`           | `[<intro>]`            | [`label`]  |
-| Reference        | `\ref{intro}`             | `[@intro]`             | [`ref`]    |
-| Citation         | `\cite{humphrey97}`       | `[@humphrey97]`        | [`cite`]   |
-| Bullet list      | `itemize` environment     | `[- List]`             | [`list`]   |
-| Numbered list    | `enumerate` environment   | `[+ List]`             | [`enum`]   |
-| Term list        | `description` environment | `[/ Term: List]`       | [`terms`]  |
-| Figure           | `figure` environment      | `figure` function      | [`figure`] |
-| Table            | `table` environment       | `table` function       | [`table`]  |
-| Equation         | `$x$`, `align` / `equation` environments | `[$x$]`, `[$ x = y $]` | [`equation`]($math.equation) |
+| Element                | LaTeX                     | Typst                  | See        |
+|:-----------------------|:--------------------------|:-----------------------|:-----------|
+| Strong emphasis        | `\textbf{strong}`         | `[*strong*]`           | [`strong`] |
+| Emphasis               | `\emph{emphasis}`         | `[_emphasis_]`         | [`emph`]   |
+| Link                   | `\url{https://typst.app}` | `[https://typst.app/]` | [`link`]   |
+| Label                  | `\label{intro}`           | `[<intro>]`            | [`label`]  |
+| Reference              | `\ref{intro}`             | `[@intro]`             | [`ref`]    |
+| Citation               | `\cite{humphrey97}`       | `[@humphrey97]`        | [`cite`]   |
+| Verbatim / code        | `\verb|print(f"{x}")|`, `verbatim` / `listing` environments | ``[`print(f"{x}")`]`` | [`raw`] |
+| Monospace / typewriter | `\texttt{mono}` | ``[`mono`]`` or `text` function | [`raw`], [`text`] |
```

Comment:
I think `text` should come first and `mono` second. The links in "See" should also go there instead of `raw`, `text`.